src/cljs/remote_manager/controller.cljs

```diff
@@ -162,28 +175,45 @@
 (defn download-file!
   [config filename]
   (let [hostname (get-in @store/state [:connection :hostname])
-        callback #(utils/save-file (get %1 "contents") filename)]
+        callback #(utils/save-file (get %3 "contents") filename)]
     (bbserver/get-file hostname config filename callback)))
 
 (defn delete-file!
   [config filename]
-  (let [hostname (get-in @store/state [:connection :hostname])
-        callback update-configs!]
-    (bbserver/delete-file hostname config filename callback)))
+  (let [hostname (get-in @store/state [:connection :hostname])]
+    (bbserver/delete-file hostname config filename update-configs!)))
 
 (defn ping
   []
   (let [hostname (get-in @store/state [:connection :hostname])
         port (get-in @store/state [:services :config])]
     (bbserver/ping hostname port #(utils/log "Pinging MachineKit"))))
 
-(defn log-state
-  "Adding whatever information you want to see for debugging here"
+; Create an update interval while the user is connected that
+; updates configs and mk-services
+(defn- start-update-configs-and-services-interval
   []
-  (utils/log (str "Services: " (:services @store/state))))
-
-(defn debug-state
-  [timeout]
-  (utils/set-interval "debug-state" log-state timeout))
-
-(debug-state 5000)
+  (utils/set-interval "update-configs-and-services"
+    #(let [{:keys [connection]} @store/state]
+      (if (:connected? connection)
+        (do
+          (update-configs!)
+          (update-mk-services!))))
+    2000))
```

Comment:
I think this can be a bit lower now that we're using HTTP instead of SSH? maybe 500-1000?